### PR TITLE
[FR] Add `Modding/Forum_modding`, `Bubble`

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the following wiki pages:

- `Modding/Forum_modding`
- `Modding/Bubble`
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/Modding/Bubble/fr.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# Bubble
+
+La **bubble** (![Icône de la bubble](/wiki/shared/icon/bubble.gif)) était une icône de message de forum utilisée par les membres des équipes de modding suivantes impliquées dans le [processus de classement des beatmaps](/wiki/Beatmap_ranking_procedure) :
+
+- [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) (*BAT*)
+- [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team) (*MAT*)
+- [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BN*)
+- [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) (*QAT*)
+
+L'étape suivante de l'[ancien système de classement](/wiki/Modding/Forum_modding) après une [proto-bubble](( /wiki/Modding/Proto-bubble), une bubble régulière signifie qu'une [beatmap](/wiki/Beatmap) est prête à être classée du point de vue du moddeur. Les beatmaps avec une bubble étaient éventuellement vérifiées par un autre membre de la BAT ou d'un BN et [classée](/wiki/Beatmap/Category#classée) ou [approuvée](/wiki/Beatmap/Category#approuvée) tant que des modifications mineures de la map étaient nécessaires.
```
Comment (DoryanR) [inline wiki/Modding/Bubble/fr.md:16]:
```suggestion
L'étape suivante de l'[ancien système de classement](/wiki/Modding/Forum_modding) après une [proto-bubble](/wiki/Modding/Proto-bubble), une bubble régulière signifie qu'une [beatmap](/wiki/Beatmap) est prête à être classée du point de vue du moddeur. Les beatmaps avec une bubble étaient éventuellement vérifiées par un autre membre de la BAT ou d'un BN et [classée](/wiki/Beatmap/Category#classée) ou [approuvée](/wiki/Beatmap/Category#approuvée) tant que des modifications mineures de la map étaient nécessaires.
```

File: wiki/Modding/Bubble/fr.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Dans la [procédure de classement du beatmap](/wiki/Beatmap_ranking_procedure) moderne, l'équivalent d'une bubble est la première nomination donnée par un [beatmap nominator](/wiki/People/The_Team/Beatmap_Nominators).
+
+## Histoire
+
+*Voir également : [Calendrier de mapping et de modding](/wiki/Mapping_and_Modding_Timeline)*
+
+Les bubbles ont été introduites par ![][flag_AU] [peppy](/wiki/People/peppy) le 29 octobre 2007, pour les "beatmaps qui sont considérés pour le jeu classé (en attendant les commentaires des modérateurs)". Le fait de transformer l'icône d'un fil de discussion d'une beatmap en une bubble était un moyen pour les membres de la BAT d'indiquer que la beatmap était de bonne qualité et qu'elle avait respecté les [critères de classement](/wiki/Ranking_Criteria).
+
+Le 3 octobre 2010, la MAT a reçu l'autorisation d'utiliser l'icône de bubble, aux côtés de la BAT. Cela a essentiellement déprécié les [proto-bubbles] (/wiki/Modding/Proto-bubbles), et les deux équipes ont principalement utilisé des bubbles ordinaires depuis.
```
Comment (DoryanR) [inline wiki/Modding/Bubble/fr.md:26]:
```suggestion
Le 3 octobre 2010, la MAT a reçu l'autorisation d'utiliser l'icône de bubble, aux côtés de la BAT. Cela a essentiellement déprécié les [proto-bubbles](/wiki/Modding/Proto-bubbles), et les deux équipes ont principalement utilisé des bubbles ordinaires depuis.
```

File: wiki/Modding/Forum_modding/fr.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+Les modifications nécessaires pour résoudre les problèmes les plus courants ont été initiées le [26 avril 2013](https://osu.ppy.sh/community/forums/topics/129625), lorsque [peppy](/wiki/People/peppy) a présenté l'idée du [nouveau système de modding](/wiki/Beatmap_Discussion) (connu ensuite sous le nom de *Modding v2*). Le [21 août 2014](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat), une nouvelle catégorie, [qualifiée](/wiki/Beatmap/Category#qualifiée), a été ajoutée. La catégorie servirait de tampon entre les beatmaps en attente et les beatmaps classés et permettrait des déranks plus fluides. De plus, une nouvelle équipe, la [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) (*QAT*), a été créée pour vérifier les beatmaps qualifiés et contrôler le processus de classement.
+
+Ensuite, le système de discussion des beatmap et le [code de conduite pour le modding et le mapping] (/wiki/Rules/Code_de_conduite_pour_le_modding_et_le_mapping#making-a-mod-post) ont été développés et mis à disposition pour restructurer le modding et en faire une expérience plus agréable pour toutes les personnes impliquées :
```
Comment (DoryanR) [inline wiki/Modding/Forum_modding/fr.md:104]:
```suggestion
Ensuite, le système de discussion des beatmap et le [code de conduite pour le modding et le mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#créer-un-mod-post) ont été développés et mis à disposition pour restructurer le modding et en faire une expérience plus agréable pour toutes les personnes impliquées :
```

File: wiki/Modding/Bubble/fr.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- Les beatmaps allant pour la catégorie [approuvée](/wiki/Beatmap/Category#approuvée) devaient avoir deux bubbles consécutives placées par deux membres de la BAT différents. Le 2 juin 2017, ce seuil a été modifié pour passer à une seule bubble.
+
+### Bubble pop
+
+Une bubble existante pourrait être **popped** :
+
+- Automatiquement, par le mappeur lui-même, s'il a mis à jour la map ou l'un de ses fichiers.
+- Manuellement, par un membre de la BAT, si la beatmap nécessite davantage de modifications et ne peut être classé dans son état actuel.
+
+Dans les deux cas, l'icône du fil de discussion de la beatmap était définie comme une popped bubble (![Icône de la Popped bubble](/wiki/shared/icon/bubble-pop.gif)). Si une bubble était éclatée manuellement, il était supposé que les membre de la BAT ou de la MAT qui avait placé l'icône s'assurerait que les modifications nécessaires étaient apportées à la map avant qu'elle ne continue dans le processus de classement.
```
Comment (Biokax) [inline wiki/Modding/Bubble/fr.md:51]:
```suggestion
Dans les deux cas, l'icône du fil de discussion de la beatmap était définie comme une popped bubble (![Icône de la Popped bubble](/wiki/shared/icon/bubble-pop.gif)). Si une bubble était éclatée manuellement, il était supposé que les membre de la BAT ou de la MAT qui avait placé l'icône s'assurerait que les modifications nécessaires étaient apportées à la map avant qu'elle ne continue dans le processus de classement.
```
```suggestion
Dans les deux cas, l'icône du fil de discussion de la beatmap était définie comme une popped bubble (![Icône de la Popped bubble](/wiki/shared/icon/bubble-pop.gif)). Si une bubble était éclatée manuellement, il était supposé que les membres de la BAT ou de la MAT qui avait placé l'icône s'assureraient que les modifications nécessaires étaient apportées à la map avant qu'elle ne continue dans le processus de classement.
```

File: wiki/Modding/Forum_modding/fr.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  - moddingv1
+  - forum-based modding
+  - old modding system
+  - modding basé sur le forum
+  - ancien système de modding
+---
+
+# Forum modding
+
+**Forum modding**, également connu sous le nom de *Modding v1*, est un nom familier du système de [modding](/wiki/Modding) et de classement basé sur le forum qui a été principalement utilisé jusqu'au [4 novembre 2017](https://osu.ppy.sh/community/forums/topics/650961?n=7), date à laquelle toutes les [beatmaps](/wiki/Beatmap) nouvelles et inactifs sont passés en [discussions d'une beatmap](/wiki/Beatmap_Discussion) par défaut.
```
Comment (Biokax) [inline wiki/Modding/Forum_modding/fr.md:14]:
```suggestion
**Forum modding**, également connu sous le nom de *Modding v1*, est un nom familier du système de [modding](/wiki/Modding) et de classement basé sur le forum qui a été principalement utilisé jusqu'au [4 novembre 2017](https://osu.ppy.sh/community/forums/topics/650961?n=7), date à laquelle toutes les [beatmaps](/wiki/Beatmap) nouvelles et inactifs sont passés en [discussions d'une beatmap](/wiki/Beatmap_Discussion) par défaut.
```
```suggestion
**Forum modding**, également connu sous le nom de *Modding v1*, est un nom familier du système de [modding](/wiki/Modding) et de classement basé sur le forum qui a été principalement utilisé jusqu'au [4 novembre 2017](https://osu.ppy.sh/community/forums/topics/650961?n=7), date à laquelle toutes les [beatmaps](/wiki/Beatmap) nouvelles et inactives sont passées en [discussions d'une beatmap](/wiki/Beatmap_Discussion) par défaut.
```

**Review by Biokax — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Biokax can you recheck please ?

File: wiki/Modding/Forum_modding/fr.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+La priorité d'une beatmap peut être augmentée par des mods recevant des kudosu ou des utilisateurs tirant des étoiles de kudosu. Dans les [discussions sur les beatmaps](/wiki/Beatmap_Discussion) modernes, la star priority est comparable au nombre de [hype](/wiki/Beatmap/Hype) d'une beatmap.
+
+### Icônes des fils
+
+La progression d'une beatmap dans la procédure de classement était indiquée par les messages postés par les membres des équipes de classement du beatmap. Chaque message avait une icône personnalisée, qui changeait également l'icône du fil de discussion du forum pour refléter son statut :
+
+- ![](/wiki/shared/icon/star.gif) **[Star](/wiki/Disambiguation/Star)** : la beatmap avait le potentiel pour devenir classé.
+- ![](/wiki/shared/icon/bubble.gif) **[Bubble](/wiki/Modding/Bubble)** : la beatmap a été vérifié par un membre de la MAT, BAT ou BN, et pourrait être classé après une autre vérification.
+- ![](/wiki/shared/icon/bubble-pop.gif) **[Bubble pop](/wiki/Modding/Bubble#bubble-pop)** : la beatmap avait des problèmes qui l'empêchaient d'être classé après avoir reçu une bulle.
```
Comment (Biokax) [inline wiki/Modding/Forum_modding/fr.md:52]:
```suggestion
- ![](/wiki/shared/icon/bubble-pop.gif) **[Bubble pop](/wiki/Modding/Bubble#bubble-pop)** : la beatmap avait des problèmes qui l'empêchaient d'être classé après avoir reçu une bubble.
```

**Review by Biokax — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]
merging with the assumption that one review was enough

**Pull request merged**